share/smack/include/pthread.h

```diff
@@ -0,0 +1,71 @@
+//
```

Comment:
Yep, we should try it out. It looks like a promising solution, but maybe there are potential issues we are not anticipating right now. I'll tell Monty to give it a try. Thanks Mike.